src/app/views/staff/staff.controller.coffee

```diff
@@ -1,20 +1,44 @@
-@App.controller 'StaffController', ($filter, $stateParams, $uibModal, toastr, MnoeUsers, MnoAppsInstances) ->
+@App.controller 'StaffController', ($log, $stateParams, $window, $uibModal, toastr, MnoeCurrentUser, MnoeUsers, MnoeSubTenants, ADMIN_ROLES) ->
   'ngInject'
   vm = this
+  vm.isSaving = false
+  vm.adminRoles = ADMIN_ROLES
 
-  vm.staff =
-    # Display staff creation modal
-    createModal: ->
-      modalInstance = $uibModal.open(
-        templateUrl: 'app/views/staff/create-staff-modal/create-staff.html'
-        controller: 'CreateStaffController'
-        controllerAs: 'vm'
-      )
+  vm.isAdmin = MnoeCurrentUser.user.admin_role == 'admin'
 
-  vm.displayed = []
+  # Get the user
+  MnoeUsers.get($stateParams.staffId).then(
+    (response) ->
+      vm.staff = response.data
+      vm.staff.subTenantName = ->
+        _.find(vm.subTenants, (subTenant) -> subTenant.id == vm.staff.mnoe_sub_tenant_id).name
+      vm.staff.adminRoleName = ->
+        _.find(vm.adminRoles, (role) -> role.value == vm.staff.admin_role).label
 
-  vm.callServer = (tableState) ->
-    console.log (tableState)
-    vm.isLoading = false
+      if(vm.staff.mnoe_sub_tenant_id)
+        MnoeSubTenants.get(vm.staff.mnoe_sub_tenant_id).then((r) -> vm.staff.subTenants = r.data.sub_tenants)
+  )
+  MnoeSubTenants.list(null, null, null).then((response) -> vm.subTenants = response.data)
```

Comment:
This will fetch all the subtenants. it may be a problem if there is a lot of subtenants.